src/main/scala/org/scalameter/BasePerformanceTest.scala

```diff
@@ -2,13 +2,17 @@ package org.scalameter
 
 
 import java.util.Date
+import org.scalameter.picklers.Pickler
 import org.scalameter.utils.Tree
+import scala.language.existentials
 import scala.util.DynamicVariable
 
 
 
 abstract class BasePerformanceTest {
 
+  type Result
```

Comment:
I think that we should have this as a type parameter on the `BasePerformanceTest` class.
This will break a lot of things, but since it's such a feature-rich release, our clients should be willing to bite the bullet.